exchanges/account/account_test.go

```diff
@@ -292,8 +296,20 @@ func TestBalanceInternalWait(t *testing.T) {
 func TestBalanceInternalLoad(t *testing.T) {
 	t.Parallel()
 	bi := &ProtectedBalance{}
-	bi.load(Balance{Total: 1, Hold: 2, Free: 3, AvailableWithoutBorrow: 4, Borrowed: 5})
+	err := bi.load(nil)
+	assert.ErrorIs(t, err, common.ErrNilPointer, "should error nil pointer correctly")
+
+	err = bi.load(&Balance{Total: 1, Hold: 2, Free: 3, AvailableWithoutBorrow: 4, Borrowed: 5})
+	assert.ErrorIs(t, err, errUpdatedAtIsZero, "should have not been loaded")
+
+	now := time.Now()
+	err = bi.load(&Balance{UpdatedAt: now, Total: 1, Hold: 2, Free: 3, AvailableWithoutBorrow: 4, Borrowed: 5})
+	assert.NoError(t, err, "should have been loaded")
```

Comment:
🦟 Nit: Since this is a unit test, and this is an NoError on the main function called, the comment can be omitted.
If it's not omitted, it should say "Must not error when X" if X is a regression case or it's not clear what obsticle has been cleared. In this case, UpdatedAt was the last error, we've now provided it, so it's obvious why it should now pass.
🚧 It also feels like this is a pre-requisit for the following, so needs to be require.
```suggestion
	require.NoError(t, err)
```